test/Sentry.Tests/ApiApprovalTests.Run.DotNet10_0.verified.txt

```diff
@@ -191,7 +207,7 @@ namespace Sentry
         void BindException(System.Exception exception, Sentry.ISpan span);
         Sentry.SentryId CaptureEvent(Sentry.SentryEvent evt, System.Action<Sentry.Scope> configureScope);
         Sentry.SentryId CaptureEvent(Sentry.SentryEvent evt, Sentry.SentryHint? hint, System.Action<Sentry.Scope> configureScope);
```

Comment:
question: Is it worth considering have more returns of `` ... potentially in a follow-up issue/PR?

- `CaptureTransaction`
- `CaptureSession`

Reply:
I don't think so. Those methods aren't user facing (they're SDK user facing). This method is an exception in that there will be some UI around it in the app and so some feedback is required for the actual user (not just the SDK user).